napari/layers/_base_layer/view/properties.py

```diff
@@ -119,7 +119,6 @@ def __init__(self, layer):
         self.setFixedWidth(250)
         self.grid_layout.setColumnMinimumWidth(0, 100)
         self.grid_layout.setColumnMinimumWidth(1, 100)
-        self.layer.selected = True
```

Comment:
oh ok that makes more sense, but why access it via `_selected` instead of `selected`, which would prompt events?

Reply:
for most things like this we only use the `_` property, but it might be worth revisiting in general as it means we might be able to clean up some of the setters in the properties files. I'd rather deal with this in follow up PRs though